api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -655,13 +656,14 @@ private Map<EmailSubstitutionField, String> workspaceAdminLockedSubstitutionMap(
   }
 
   private Map<EmailSubstitutionField, String> publishUnpublishWorkspaceSubstitutionMap(
-      DbWorkspace workspace, String publishCategory, String supportEmail) {
+      DbWorkspace workspace, DbUser user, String supportEmail) {
     return new ImmutableMap.Builder<EmailSubstitutionField, String>()
         .put(EmailSubstitutionField.HEADER_IMG, getAllOfUsLogo())
         .put(EmailSubstitutionField.ALL_OF_US, getAllOfUsItalicsText())
+        .put(EmailSubstitutionField.FIRST_NAME, user.getGivenName())
+        .put(EmailSubstitutionField.LAST_NAME, user.getFamilyName())
         .put(EmailSubstitutionField.WORKSPACE_NAME, workspace.getName())
         .put(EmailSubstitutionField.WORKSPACE_NAMESPACE, workspace.getWorkspaceNamespace())
-        .put(EmailSubstitutionField.PUBLISH_CATEGORY, publishCategory)
```

Comment:
don't need this anymore because we only email for Community.  I'm hard-coding this to ensure we don't accidentally send anything else.